package/expo-package/src/handlers/Video.tsx

```diff
@@ -1,16 +1,29 @@
-import { VideoComponent } from '../optionalDependencies';
+import React, { useEffect } from 'react';
+
+import { AudioComponent, VideoComponent } from '../optionalDependencies';
 
 export const Video = VideoComponent
-  ? ({ onPlaybackStatusUpdate, paused, resizeMode, style, uri, videoRef }) => (
-      <VideoComponent
-        onPlaybackStatusUpdate={onPlaybackStatusUpdate}
-        ref={videoRef}
-        resizeMode={resizeMode}
-        shouldPlay={!paused}
-        source={{
-          uri,
-        }}
-        style={[style]}
-      />
-    )
+  ? ({ onPlaybackStatusUpdate, paused, resizeMode, style, uri, videoRef }) => {
+      // This is done so that the audio of the video is not muted when the phone is in silent mode for iOS.
+      useEffect(() => {
+        const initializeSound = async () => {
+          await AudioComponent.setAudioModeAsync({
+            playsInSilentModeIOS: true,
+          });
+        };
+        initializeSound();
+      });
```

Comment:
useEffect without dependencies? Is this intended?

Reply:
Ops, I missed adding it. Stupid mistake. A [] should be there.